template.distillery/demo_calendar.eliom

```diff
@@ -21,6 +21,18 @@ let%server service =
 let%client service = ~%service
 
 (* A reactive value containing the currently selected date *)
+(* NOTE: in this example, we define a shared signal on the server side. It can
+   be accessed when the server generates the first page (so, declaring it
```

Comment:
Not updatable since it's wrapped in a `Eliom_shared.Value.t`? Is it worth mentioning it there? Maybe parts of this should be moved to the corresponding documentation page...